build-system/release-workflows/promote-lts.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const {createVersioningUpdatePullRequest} = require('./promote-job');
+const {log} = require('../common/logging');
+const {runReleaseJob} = require('./release-job');
+
+/**
+ * @fileoverview Script that promotes the current Stable traffic channel to LTS.
+ */
+
+const jobName = 'promote-stable.js';
+
+runReleaseJob(jobName, async () => {
+  const dayOfMonth = new Date().getUTCDate();
+  if (!(8 <= dayOfMonth && dayOfMonth <= 14)) {
```

Comment:
One of the `<=`s needs to be flipped

Reply:
if NOT (dayOfMonth between 8 and 14) - is that wrong?